diablo/api/course_controller.py

```diff
@@ -146,6 +148,14 @@ def unschedule():
     db.session.execute(
         Scheduled.__table__.delete().where(and_(Scheduled.term_id == term_id, Scheduled.section_id == section_id)),
     )
+
+    kaltura_schedule_id = scheduled['kalturaScheduleId']
+    try:
+        Kaltura().delete_scheduled_recordings(kaltura_schedule_id)
+    except (KalturaClientException, KalturaException) as e:
+        app.logger.error(f'Failed to delete Kaltura schedule: {kaltura_schedule_id}')
+        app.logger.exception(e)
```

Comment:
Willa is in favor of programmatically deleting Kaltura series, as seen above. The Jira: https://jira.ets.berkeley.edu/jira/browse/DIABLO-282